decidim-core/config/locales/en.yml

```diff
@@ -96,6 +96,7 @@ en:
         error: There was a problem deleting your account.
         success: Your account was successfully deleted.
       show:
+        available_locales_helper: The language in which you'll be using to browse Decidim and receive email notifications
```

Comment:
This seems misleading to me, because this value only seems to be used in the emails, not in the browsing of the website.

Reply:
Fixed now. Thanks